core/shared/platform/common/posix/posix_thread.c

```diff
@@ -387,6 +387,10 @@ static uint32
 touch_pages(uint8 *stack_min_addr, uint32 page_size)
 {
     uint8 sum = 0;
+
+    if (stack_min_addr == 0)
```

Comment:
No need to add check here, as you have added check in Line 418, here it won't be 0.

Reply:
Ok. I agree.